concourse/book-scan/script.bash

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
i think you should use the docker image built for CI that has all this installed already

Reply:
Ah I see what you mean... I also probably need to figure out why the runs on concourse have so many errors...